Readme Update

ShowMeYourSkil opened the pull request:
Good morning,

I have upgraded your readme. Now it is more informative, nicer and just better. 
Since I'm not you and don't know exactly what you want it to say, I wrote in what I think you want to read. If something is doff or missing you can update it after the pull request.

ShowMeYourSkil committed:
Create Ansible-ressource

ShowMeYourSkil committed:
Rename Ansible-ressource to ansible-ressource.md

ShowMeYourSkil committed:
Finished my Oracle Virtualbox articel

Rename ansible-ressource -> virtualbox

ShowMeYourSkil committed:
Update virtualbox.md

ShowMeYourSkil committed:
Copy in directory /posts

Add a link to my first article

ShowMeYourSkil committed:
Add topic 'How to start a Virtual machine'

ShowMeYourSkil committed:
Create PULL_REQUEST_TEMPLATE.md

ShowMeYourSkil committed:
test

ShowMeYourSkil committed:
Update README.md

ShowMeYourSkil committed:
Finish

ShowMeYourSkil committed:
Delete README.md

ShowMeYourSkil committed:
Delete PULL_REQUEST_TEMPLATE.md

ShowMeYourSkil committed:
Create README.md

mtorials commented:
Ouh here is still the virtualbox aricle in!

ShowMeYourSkil committed:
Delete virtualbox.md

ShowMeYourSkil commented:
> Ouh here is still the virtualbox aricle in!

Ups. My new branch struktur for all pull_request is new :D i have already remove the article for you

mtorials commented on README.md:
```diff
@@ -1,16 +1,42 @@
+<h1>https://mt32.net</h1>
+<h2>The blog for all computer interested</h2>
+
+ Netlify  
```
mt32.net is no longer deployed with netlify

ShowMeYourSkil replied:
Okey thank you. I want to remove netlify!

mtorials commented on README.md:
```diff
@@ -1,16 +1,42 @@
+
+# About mt32.net
+
+`mt32.net` is running by @mtorials and @universum two hobby programmer. 
```
I am the only one working on this....

ShowMeYourSkil replied:
i was use the source on your about page. You should update this site.

ShowMeYourSkil replied:
I was writen from your about source. https://mt32.net/about. I will remove universum.

mtorials commented on README.md:
```diff
@@ -1,16 +1,42 @@
+Do you want make a article! Nice! Please read THIS file when you're new and do not know how.
+
+# Netifly
+
```
again this is no longer the case

ShowMeYourSkil replied:
Okey. I will remove them.

ShowMeYourSkil committed:
remove lines. 

Remove @universum, remove netlify text and status table, Update ,,How to write a article?", add global

ShowMeYourSkil reviewed (commented):
@mtorials i have meet your wishes. Now it is okey for you?
